arm-sql/2015-05-01-preview/examples/DatabaseRecommendedActionGet.json

```diff
@@ -0,0 +1,62 @@
+{
+    "parameters": {
+        "subscriptionId": "741fd0f5-9cb8-442c-91c3-3ef4ca231c2a",
+        "resourceGroupName": "workloadinsight-demos",
+        "serverName": "misosisvr",
+        "databaseName": "IndexAdvisor_test_3",
+        "advisorName": "CreateIndex",
+        "recommendedActionName": "IR_[CRM]_[DataPoints]_4821CD2F9510D98184BB",
```

Comment:
Recommended action names are always like that in production, so I would not simplify it